### DISCUSSION TITLE
Future Plans with the project and new updates

### DISCUSSION THREAD

**Discussion by 777Sev777** [lightbeam24/CustomTube#184] [Q&A]
Really excited for the upcoming update. I was just wondering, is an update to add the old school comments sections for 2008-2012 layouts on the table for the near future? - Or are the next few updates solely focusing on improving parody with the old designs and fixing inaccuracies and bugs?

Thanks for keeping up with this plugin, its great work and def one of my fav extensions.

**Comment by lightbeam24** [comment]
Currently, the plan going forward is to turn CustomTube into a pseudo-frontend by replacing as much of the site's HTML as possible. This not only means it will look more accurate, but it will also *feel* more accurate (it's hard to explain, but you'll know when you use it). However, comments in particular are not part of this plan as of now (due to some technical reasons, it would be risky to do right now).
